openedx/core/djangoapps/courseware_api/serializers.py

```diff
@@ -93,7 +93,6 @@ class CourseInfoSerializer(serializers.Serializer):  # pylint: disable=abstract-
     license = serializers.CharField()
     media = _CourseApiMediaCollectionSerializer(source='*')
     name = serializers.CharField(source='display_name_with_default_escaped')
-    number = serializers.CharField(source='display_number_with_default')
     offer = serializers.DictField()
     org = serializers.CharField(source='display_org_with_default')
```

Comment:
Not org too? The frontend PR drops that, right?

Reply:
Yes! Good catch, thanks.